src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/RankSumTest.java

```diff
@@ -26,7 +26,7 @@ public RankSumTest(final boolean useDithering){
     }
 
     public RankSumTest(){
-        this(true);
+        this(false);
```

Comment:
because tests must be reproducible. https://github.com/broadinstitute/gatk/issues/1881 captures the need to make `dither` into a parameter